spamFilter.go

```diff
@@ -10,12 +10,46 @@ import (
 	"math"
 	"sort"
 )
-/*
-func fileToString() string {
-	reader := bufio.NewReader(os.Stdin)
-   	fmt.Print("Enter the name of the file you want to read: ")
-	fileName, _ := reader.ReadString('\n')
-	fileName = strings.TrimSpace(fileName)
+
+func buildCorpus(dirName string) (string, int) {
+	fileSlice, err := ioutil.ReadDir(dirName)
```

Comment:
The practice of including information about a variable's type in its variable name is referred to as "hungarian notation" by some people and inspires varying opinions - some people find it helpful, but most code bases I work on don't include type information in the name, instead preferring to name the type itself if the type is important.

Here you might be able to get away with "files" with no loss of clarify.

Reply:
I used Hungarian notation mostly to help myself remember exactly what types all the io and os functions returned and not get confused when converting their types. I suppose someone more familiar with the language would find it clumsy or annoying. I'll guess I'll try avoiding Hungarian notation in the future and instead helping myself with comments, which I assume is more conventional anyway.

Reply:
Totally fine. Over time it'll become easier to remember what the types are by their usages (and in your editor and tooling, which you don't have set up yet).